node/Cargo.toml

```diff
@@ -44,6 +44,9 @@ version = "1"
 [dependencies.num_cpus]
 version = "1"
 
+[dependencies.once_cell]
```

Comment:
Can we use the now stabilised std instead?

Reply:
I've considered this before, but only a small subset of the `OnceCell` has been stabilized so far - we would still need to use `once_cell` for now.